stestr/commands/run.py

```diff
@@ -363,6 +366,13 @@ def run_command(config='.stestr.conf', repo_type='file',
                ">= 0 must be used.\n" % concurrency)
         stdout.write(msg)
         return 2
+    if dynamic:
+        if sys.version_info()[0] < 3 or sys.version_info[1] < 5:
```

Comment:
For obvious reasons this causes:

    TypeError: 'sys.version_info' object is not callable

```suggestion
        if sys.version_info[0] < 3 or sys.version_info[1] < 5:
```

Reply:
Oops, good catch thanks